lib/parsers/gjs-gts-parser.js

```diff
@@ -291,6 +291,16 @@ function preprocessGlimmerTemplates(info, code) {
       n.type = `Glimmer${n.type}`;
       allNodeTypes.add(n.type);
     }
+    // ast should not contain comment nodes
+    for (const comment of comments) {
+      const parentBody = comment.parent.body || comment.parent.children;
+      const idx = parentBody.indexOf(comment);
+      parentBody.splice(idx, 1);
+      // comment type can be a block comment or a line comment
+      // block comments start with <!--- or  {{!--, line comments with <!-- or {{!
+      comment.type = /^({{!--|<!---)/.test(code.slice(...comment.range)) ? 'Block' : 'Line';
```

Comment:
```suggestion
      comment.type = /^({{!|<!---)/.test(code.slice(...comment.range)) ? 'Block' : 'Line';
```

Reply:
i found out that block comment works for eslint in all cases

Reply:
yay!